deployment-strategies/quarkus-serverless/pom.xml

```diff
@@ -17,9 +17,6 @@
     <name>Quarkus OpenShift TS: Deployment Strategies Serverless: Quarkus</name>
 
     <properties>
-        <!-- this could also be set in application.properties, but this is much closer to what users actually do -->
-        <!-- We do not use <quarkus.kubernetes.deploy>true</quarkus.kubernetes.deploy> intentionally, just the build -->
-        <quarkus.container-image.build>true</quarkus.container-image.build>
         <quarkus.kubernetes-client.trust-certs>true</quarkus.kubernetes-client.trust-certs>
```

Comment:
This should probably also be removed, then? (And possibly added to `UsingQuarkusPluginDeploymentStrategy`.)

Reply:
Well spotted! Moving to UsingQuarkusPluginDeploymentStrategy.